cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -1374,22 +1387,99 @@ private KafkaVersionChange getKafkaVersionChange(StatefulSet kafkaSts) {
          * There is one special case of scaling from standalone (single one) Zookeeper pod.
          * In this case quorum cannot be preserved.
          */
-        Future<ReconciliationState> zkScaleUpInSmallSteps() {
+        Future<ReconciliationState> zkScalingSetup() {
```

Comment:
I guess we should probably fix the JavaDoc?

Reply:
Good point